src/main/java/Event.java

```diff
@@ -0,0 +1,25 @@
+public class Event extends Task {
+    protected String at;
```

Comment:
Maybe the attribute name "at" can be changed to a noun like "date" or "eventDate" to enhance readability?